sig-and-wg/sig-core/proposals/authorization-proxy.md

```diff
@@ -0,0 +1,28 @@
+# Authorization proxy
+
+
+## Abstract
+
+It is not possible to install kyma project on managed clusters like Google Kubernetes Engine (GKE) or Azure Kubernetes Service (AKS). To enable kyma-project adoption it has to run on any infrastructure supporting Kubernetes. Kyma requires access to kube-apiserver configuration and uses some kubernetes alpha features, that is **not allowed** in many managed clusters. Even if some providers allow for such configuration, it has to happen during cluster provisioning, what makes kyma installation process more complex.
+
+
+## Problems to solve
+
+1. The kube-apiserver component must be configured to verify tokens coming from an external OIDC provider (coming with Kyma). Usually, it is done during the installation by providing additional parameters such as **oidc-issuer-url**, **oidc-client-id**, **oidc-username-claim**, to name a few. These parameters cannot be set in GKE or AKS.
```

Comment:
Remark: Make "cannot be set" more explicit and use bold.

Reply:
done